components/molecules/PopularCategoryCard.js

```diff
@@ -1,34 +1,45 @@
 import PropTypes from "prop-types";
 import Image from "next/image";
+import { LibraryIcon } from "@heroicons/react/solid";
 
 export function PopularCategoryCard(props) {
   return (
     <div
-      className={`md:shadow-md h-auto min-h-96 w-full rounded-md border-2 pl-3 pr-3 ${
+      className={`flex flex-col items-center md:shadow-md w-40 h-40 rounded-full border-2 hover:shadow-cards bg-white overflow-clip overflow-hidden${
         props.selected ? "border-blue-500" : ""
       }`}
       onClick={() => props.onClick(props.id)}
     >
       {props.imgSource ? (
-        <div className="py-5">
-          <div style={{ position: "relative", width: "100%", height: "5rem" }}>
+        <div className=" mt-4 mb-2 p-1 bg-gray-100  rounded-md">
+          <div className="aspect-w-1 aspect-h-1 w-16 rounded-md  ">
             <Image
               src={props.imgSource}
               alt={props.imgAltText}
               layout="fill"
+              className="rounded-md object-cover"
             ></Image>
           </div>
         </div>
       ) : (
-        ""
+        <div className=" mt-4 mb-2 p-1 bg-gray-100  rounded-md">
+          <div className="aspect-w-1 aspect-h-1 w-8 rounded-md  ">
+            <LibraryIcon
+              className=" text-gray-500"
+              alt="support-icon"
+            ></LibraryIcon>
+          </div>
+        </div>
       )}
-      <div className="my-5">
+      <div className="mb-5">
         <small className="text-gray-500 uppercase">{props.type}</small>
         <a href="#catalog">
-          <h2 className="text-h3">{props.title}</h2>
+          <h2 className="px-2 font-display text-base text-center overflow-ellipsis line-clamp-2">
+            {props.title}
+          </h2>
         </a>
       </div>
-      <p className="pb-4">{props.description}</p>
+      {/* <p className="pb-4">{props.description}</p> */}
```

Comment:
We can remove this line :)

Reply:
I took out the description and the type as they are no longer part of the design as they were originally. The props are still there so they could be added or used in some why if the design changes again.